### ISSUE TITLE
Incorrect focus order

### ISSUE THREAD

**Issue by anna-guo-level-access** [anna-guo-level-access/a#27]
**Link to finding**  
https://tenantb.localhost-dashboard.essentialaccessibility.com/manual-evaluations/69011c3564c5307d99a4e94e/results/701/view?skipScanStandards=true&status=toReview,open,fixed,partiallyFixed,notFixed,notReproducible,cannotBeFixed,validationRequested&linkedPropertyData=68faae85659492d7b1fe4d63%7C68faadce659492d7b1fe4824

**Severity**  
High

**URL**  
https://www.uab.edu/home/

**Actual result**  
The My Account login does not have a logical navigation sequence using only the keyboard.

It is important to provide a logical navigation sequence for keyboard-only user with motor disabilities or non-visual users that can't use the mouse to navigate the page in order for them to fully understand the content.

**Steps to reproduce**  
1. Go to the URL.
2. Navigate through the page using the keyboard until you reach the My account login.
3. Notice that the focus order is not the same as the one being displayed on the screen.

**Issue frequency**  
Global

**Screen**  
Product

**Screenshot**  
https://download.essentialaccessibility.com/images/demo/issue2.png

**WCAG Success criteria**  
2.4.3

**WCAG Conformance level**  
A

**Effort**  
High

**Instances**  
None provided

**Disabilities Affected**  
Visual, Cognitive, Physical

**Recommendation**  
Place the HTML <elements> within the code in a logical order, so that the default tab order follows that sequence.

Refer to:
https://www.w3.org/WAI/WCAG21/Techniques/general/G59.html

**Link to EAP**  
https://tenantb.localhost-dashboard.essentialaccessibility.com/projects/6a3c613b43450165564cc432/tasks/6a47ca8fc4c832192955b575

**Task in the Level Access Platform**  
https://tenantb.localhost-dashboard.essentialaccessibility.com/my-tasks/6a3c613b43450165564cc432/6a47ca8fc4c832192955b575

**Comment by anna-guo-level-access** [comment]
lap2

Posted by Anna Guo via Level Access

**Comment by anna-guo-level-access** [comment]
..lap

Posted by Anna Guo via Level Access

Posted by Anna Guo via Level Access

**Comment by anna-guo-level-access** [comment]
git1

Posted by Anna Guo via Level Access

**Comment by anna-guo-level-access** [comment]
git 44

Posted by Anna Guo via Level Access

**Comment by anna-guo-level-access** [comment]
testinggg

Posted by Anna Guo via Level Access

Posted by Anna Guo via Level Access

**Comment by anna-guo-level-access** [comment]
Comment deleted on GitHub Issues

Posted by Anna Guo via Level Access

**Comment by anna-guo-level-access** [comment]
Task pushed to GitHub Issues by Anna Guo from Level Access Platform.

Posted by Anna Guo via Level Access

**Comment by anna-guo-level-access** [comment]
Task pushed to GitHub Issues by Anna Guo from Level Access Platform.

Posted by Anna Guo via Level Access

Posted by Anna Guo via Level Access

**Comment by anna-guo-level-access** [comment]
Task pushed to GitHub Issues by Anna Guo from Level Access Platform.

Posted by Anna Guo via Level Access
